### ISSUE TITLE
When is glass not transparent 1 tile away?

### ISSUE THREAD

**Issue by Anaglyphic** [CleverRaven/Cataclysm-DDA#48694]
# Describe the bug

I was posting in discussions about a strange issue not being able to pick up canned food without emptying the cans first, but you can't actually do that.  https://github.com/CleverRaven/Cataclysm-DDA/discussions/48621 That was apparently related to savegame conversion and the "pockets" format of the new version - easy to manually fix but the savegame wasn't that important.

However in the process of complaining abou ti, I spotted another issue, with glass not being transparent - I cannot drive cars, everything's obscured as though it is solid. But _only_ when you're up against it. Climb in the back seat, you can see out the windshield; climb in the passenger side and you can see out the driver's side. It's bizarre.

It extends to all glass, even _opened windows_

With further pondering it's acting actually more like it's high-wall glass, like a 2nd floor or a church main chapel where from the top-down you see the stained glass panel, but can't see through it, can't smash it, etc. But back up and you can? It's SORT OF like that. But not really. I made an animated gif of screenshots I was going to post on reddit to see if anyone had any idea what's up, I will post it also in that discussions thread.

I haven't checked with the zombies if I'm actually obscured or it's just a visual artefact. I expect it's the former, if the code thinks it's not transparent then they're not going to see me, but I will verify that and edit.

# Steps To Reproduce

Just play. It eventually happens. But only to me? Crazy.

# Expected behavior

Glass stays transparent when you're beside it, makes things like driving possible.

# Screenshots

Plenty of screenshots stacked in an animated GIF I made for reddit over here: https://github.com/CleverRaven/Cataclysm-DDA/discussions/48621

# Versions and configuration

A few versions, but all are Narc's Jenkins autobuilds up to and including the last 04/15 10395-g4525b4f, and regular 0.E-3 
   
OS is linux 5.11.15/.17 and 5.12, as well as even tested a Windows version via wine 6.3 all exhibit this behaviour if I cp the world and load the save.

 Ingame language: English
 
Mods loaded: 

Normal play:  dda, no_npc_food, stats_through_kills, StatsThroughSkills, alt_map_key, sees_player_retro
Now that I'm testing:  dda, no_npc_food

**Comment by ZhilkinSerg** [comment]
Stand up and do not crouch?

**Comment by Zireael07** [comment]
@ZhilkinSerg: If that's indeed the issue, then it means crouch mode is not made clear enough by the UI....

**Comment by ZhilkinSerg** [comment]
Movement mode indicator is just a single character (with color coding). It is probably not feasible to put more info to sidebar. Maybe we need to put more detailed info to a new full screen menu, to existing @-menu or to move mode change menu.

**Comment by Anaglyphic** [comment]
lol goddamnit. This is 100% on me. Close this. Delete this. Burn this. :rofl: 

That is it, and it is my fault for not noticing it, I usually don't care what the movement costs me unless it's combat, and since once you press f it overlays the sidebar anyway, and your per-style movement costs are right there...

I assumed that if you ever ran, it would auto-disable and move back to walk, because we're so used to 3D FPS mouse gaming and using shift to sprint... I just assumed everytime I toggled it it goes back to walk.

Just like _of course_ every time you press " it is highlighted on w walk... you're not ON walk, look to the sidebar... it's a console game, that's just CDDAs way, never even noticed, of course the game with DF levels of micromanagement would require you to constantly and always manually specify your movement.

That you're still crouched exiting vehicle mode, sure engines and armour plating is heavy but exiting sleep? I guess some people in the apocalypse sleep curled up in a ball and roll out of bed that way, literally.

But no it's clear enough, no work needs to be done - maybe making it an easy-to-search-for FAQ answer - if things don't look right, maybe try standing up.

Feature request: I have played ~28 hours crouched, an entire in-game week, and never once noticed (xxxC) I should get a "duckwalker" achievement.

Well, if you ever decide to add a "verbose" message toggle, to handhold, I would love an intermittent "you are <movement>." every 60 moves or every hour on the hour, but of course now that I know I'll never forget it.
